docs/HOWTO.md

```diff
@@ -64,6 +64,7 @@ aws:
   resource_files:  # this allows to automatically upload custom config + frameworks to the running instance (benchmark files are always uploaded).
     - '{user}/config.yaml'
     - '{user}/frameworks.yaml'
+    - '{user}/constraints.yaml'
```

Comment:
Just double-checking that this is deliberate since the remainder of the PR is unrelated.

Reply:
Yes, it's only useful for users applying custom constraints in AWs mode, but this was forgotten.